pkg/system/reconciler.go

```diff
@@ -205,6 +209,12 @@ func NewReconciler(
 	r.SecretServer.StringData["jwt"] = util.RandomBase64(16)
 	r.SecretServer.StringData["server_secret"] = util.RandomHex(4)
 
+	// set noobaa root master key secret
+	r.NooBaaRootSecret.Data["cipher_iv"] = util.RandomBytes(16)
+	r.NooBaaRootSecret.Data["cipher_key"] = util.RandomBytes(32)
+	r.NooBaaRootSecret.StringData["cipher_type"] = "aes-256-gcm"
+	r.NooBaaRootSecret.StringData["description"] = "Root Master Key"
+	r.NooBaaRootSecret.StringData["disabled"] = "false"
```

Comment:
So this means we need multiple master-keys in this secret in order to do rotation.
Either we can write in the secret a single key with a json payload, or we can make some prefix notation to specify which item we define like:
```
	r.NooBaaRootSecret.StringData["0.cipher_type"] = "aes-256-gcm"
	r.NooBaaRootSecret.StringData["0.description"] = "Root Master Key 2020"
	r.NooBaaRootSecret.StringData["0.disabled"] = "true"

	r.NooBaaRootSecret.StringData["1.cipher_type"] = "aes-256-gcm"
	r.NooBaaRootSecret.StringData["1.description"] = "Root Master Key 2021"
	r.NooBaaRootSecret.StringData["1.disabled"] = "false"
```

Reply:
I'll handle rotation in the next PR so for now I think the current solution is ok